### PR TITLE
:sparkles:  Initial porting the shellscript to go

### PR DESCRIPTION
**Pull request by naveensrinivasan**
Porting the shellscript to go

related #79

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+func initalizeENVVariables() error {
+	/*
+	 https://docs.github.com/en/actions/learn-github-actions/environment-variables
+	   GITHUB_EVENT_PATH contains the json file for the event.
+	   GITHUB_SHA contains the commit hash.
+	   GITHUB_WORKSPACE contains the repo folder.
+	   GITHUB_EVENT_NAME contains the event name.
+	   GITHUB_ACTIONS is true in GitHub env.
+	*/
+	if err := os.Setenv("ENABLE_SARIF", "1"); err != nil {
```
Comment (laurentsimon) [inline main.go:43]: wan to create a set of errors we can re-use?

**Review by laurentsimon — APPROVED** [review]
Thanks, looking good

File: main.go (hunk 1 above)
Comment (naveensrinivasan) [reply main.go:43]: Can I do it as part of the next PR?

Comment (laurentsimon) [reply main.go:43]: sure!

Comment (azeemshaikh38) [reply main.go:43]: +1. Will be useful to have re-usable fns here. Maybe consider a `map[string]string` which can be iterated over to set the envVars. Can do this in next PR.

**Review by azeemshaikh38 — APPROVED** [review]
Thanks @naveensrinivasan!

File: main.go (hunk 1 above)
Comment (naveensrinivasan) [reply main.go:43]: Sounds good. Thanks, guys!

**Pull request merged**